packages/backend/src/yeomanui.ts

```diff
@@ -248,6 +248,10 @@ export class YeomanUI {
     process.chdir(this.initialCwd);
   }
 
+  private resetHeaderTitle(): void {
+    void this.rpc.invoke("setHeaderTitle", [undefined, undefined]);
```

Comment:
```suggestion
    void this.youiEvents.setAppWizardHeaderTitle(undefined);
```

Reply:
We should not expose that as only interaction allowed is through the API.